algorithms/lazy_vlm/HKVCacheFast.cpp

```
@@ -0,0 +1,127 @@
+// Copyright (c) MLLM Team.
+// Licensed under the MIT License.
+
+#include <cstring>
+#include <mllm/mllm.hpp>
+#include <mllm/core/Tensor.hpp>
+#include <mllm/utils/Common.hpp>
+#include <mllm/utils/UnsafeMacros.hpp>
+
+#include "HKVCacheFast.hpp"
+
+HKVCacheFast::HKVCacheFast(int32_t max_cache_length, int32_t layer_nums, int32_t q_heads, int32_t kv_heads, int32_t kv_dims,
```

Clang (14.0.6)

[error] constructor does not initialize these fields: q_heads_, kv_heads_, kv_dims_, layer_nums_, max_cache_length_, device_type_, k_dtype_, v_dtype_, k_cache_, v_cache_, h_cache_, current_seq_cnt_

(cppcoreguidelines-pro-type-member-init,-warnings-as-errors)